### PR TITLE
chore(quantic): skip authorize outside CI

### PR DESCRIPTION
**Pull request by louis-bompart**
Following @mmitiche advice, but automatedish

https://coveord.atlassian.net/browse/KIT-3044

### CODE REVIEW

File: packages/quantic/scripts/build/deploy-community.ts [hunk 1]
```diff
@@ -312,7 +312,9 @@ async function deleteScratchOrg(
 
   try {
     let communityUrl = '';
-    runner.add(async (log) => await authorizeDevOrg(log, options));
+    if (process.env.CI) {
+      runner.add(async (log) => await authorizeDevOrg(log, options));
+    }
```
Comment (mmitiche) [inline packages/quantic/scripts/build/deploy-community.ts:317]: in line `323` we re do the same step:
```
runner.add(async (log) => await authorizeDevOrg(log, options));
```
I don't think it's necessary, is it?

File: packages/quantic/scripts/build/deploy-community.ts [hunk 2]
```diff
@@ -312,7 +312,9 @@ async function deleteScratchOrg(
 
   try {
     let communityUrl = '';
-    runner.add(async (log) => await authorizeDevOrg(log, options));
+    if (process.env.CI) {
```
Comment (mmitiche) [inline packages/quantic/scripts/build/deploy-community.ts:315]: Kinda the same thing but  in this file we already have a function called `isCi`.
```suggestion
    if (isCi()) {
```

File: packages/quantic/scripts/build/deploy-community.ts (hunk 1 above)
Comment (louis-bompart) [reply packages/quantic/scripts/build/deploy-community.ts:317]: Yeah same you're right.

File: packages/quantic/scripts/build/deploy-community.ts [hunk 3]
```diff
@@ -312,13 +312,17 @@ async function deleteScratchOrg(
...
+    }
     if (options.deleteOldOrgs) {
       runner.add(async (log) => {
         await deleteOldOrgs(log, options);
       });
     }
-    runner.add(async (log) => await authorizeDevOrg(log, options));
+    if (isCi()) {
+      runner.add(async (log) => await authorizeDevOrg(log, options));
+    }
```
Comment (mmitiche) [inline packages/quantic/scripts/build/deploy-community.ts:325]: I mean is authorizing the dev org  the for the second time really necessary?
```suggestion
```

**Review by mmitiche — APPROVED** [review]

File: packages/quantic/scripts/build/deploy-community.ts (hunk 3 above)
Comment (louis-bompart) [reply packages/quantic/scripts/build/deploy-community.ts:325]: I reckon yes because deleteOldOrgs do connect to other org I reckon.

Comment (louis-bompart) [reply packages/quantic/scripts/build/deploy-community.ts:325]: Could try without tho, yolo.
If it works gg

**Pull request merged**